src/base/types.h

```diff
@@ -42,5 +42,6 @@ enum class ShutdownAction
 };
 
 typedef QMap<QString, QString> QStringMap;
+using QByteArrayMap = QMap<QByteArray, QByteArray>;
```

Comment:
http://stackoverflow.com/questions/10747810/what-is-the-difference-between-typedef-and-using-in-c11#comment25929444_10748056
seems `using` syntax is extended because of problems working with template.